### DISCUSSION TITLE
Is it possible to not have to use a baseline?

### DISCUSSION THREAD

**Discussion by Snolandia** [microsoft/vcpkg#34400] [Q&A] [answered]
Have been trying to figure out how to get my packages managed without them needing a baseline file. Ive spent quite some time searching for a solution but so far have come up with nothing. Is it just a requirement that a repo has a baseline file in it or am i missing a setting somewhere?

**Comment by autoantwort** [comment]
You don't need a baseline file. You only need a baseline if you want to use versioning.

**Reply by Snolandia** [reply]
Would you be able to link the documentation or point me in the direction for it? The only documentation i could find used baselines and everytime i try without baselines i get errors, so im obviously missing something.

**Reply by autoantwort** [reply]
For example https://learn.microsoft.com/en-us/vcpkg/consume/manifest-mode

**Reply by Snolandia** [reply]
I only see that referencing how to add dependencies from the public registry, where as I am trying to add from git repos.

**Reply by autoantwort** [reply, marked as answer]
Ah if you want to use a custom git registry you have to use a baseline.

**Reply by Snolandia** [reply]
Bummer, since that basically means i have to manage all the packages manually then....
